core/dbt/cli/main.py

```diff
@@ -73,6 +76,14 @@ def cli(ctx, **kwargs):
     if flags.RECORD_TIMING_INFO:
         ctx.with_resource(profiler(enable=True, outfile=flags.RECORD_TIMING_INFO))
 
+    project_root = flags.PROJECT_DIR if flags.PROJECT_DIR else os.getcwd()
+    cli_vars = parse_cli_vars(getattr(flags, "vars", "{}"))
+
+    # TODO need profile to exisit
+    profile = None
+    # project need profile to render because it requires knowing Target
+    ctx.obj["project"] = load_project(project_root, profile, cli_vars)
```

Comment:
Good call-- might as well start stuffing that context obj with all the things we want to pass on.